### PR TITLE
more ling fixes

### PR DESCRIPTION
**Pull request by whateverusername0**
## About the PR
god
thanks durk

:cl:
- tweak: When killing a changeling, his equipment e.g. armblade gets deleted
- fix: Fixed chemical regeneration. Thanks durkk.

### CODE REVIEW

**Comment by Verbalase** [comment]
shitcode

**Comment by whateverusername0** [comment]
> shitcode

real

File: Content.Server/Goobstation/Changeling/ChangelingSystem.cs [hunk 1]
```diff
@@ -508,6 +511,17 @@ public bool TryTransform(EntityUid target, ChangelingComponent comp, bool sting
...
+    {
+        // yanderedev type shit
+        EntityManager.DeleteEntity(comp.ShieldEntity);
+        EntityManager.DeleteEntity(comp.ArmbladeEntity);
+        EntityManager.DeleteEntity(comp.ArmorEntity);
+        EntityManager.DeleteEntity(comp.ArmorHelmetEntity);
+        EntityManager.DeleteEntity(comp.SpacesuitEntity);
+        EntityManager.DeleteEntity(comp.SpacesuitHelmetEntity);
+        PlayMeatySound(target, comp);
+    }
```
Comment (misandrie) [inline Content.Server/Goobstation/Changeling/ChangelingSystem.cs:524]: Kira wasnt lying

Comment (Verbalase) [reply Content.Server/Goobstation/Changeling/ChangelingSystem.cs:524]: > Kira wasnt lying

guys see i know code fr

**Pull request merged**